<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>929</width>
    <height>743</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGraphicsView" name="canvas">
        <property name="cursor" stdset="0">
         <cursorShape>CrossCursor</cursorShape>
        </property>
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="tabletTracking">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item row="1" column="0" colspan="3">
         <widget class="QPushButton" name="draw">
          <property name="text">
           <string>Отобразить</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1" colspan="2">
         <widget class="QDoubleSpinBox" name="z_min_sb">
          <property name="minimum">
           <double>-100.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="value">
           <double>-2.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="3">
         <widget class="Line" name="line">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Z_max&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1" colspan="2">
         <widget class="QDoubleSpinBox" name="x_min_sb">
          <property name="minimum">
           <double>-100.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="value">
           <double>-2.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;X_step&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0" colspan="3">
         <widget class="Line" name="line_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="9" column="1" colspan="2">
         <widget class="QDoubleSpinBox" name="z_step_sb">
          <property name="minimum">
           <double>-100.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="value">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="8" column="1" colspan="2">
         <widget class="QDoubleSpinBox" name="z_max_sb">
          <property name="minimum">
           <double>-100.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="value">
           <double>2.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="5" column="1" colspan="2">
         <widget class="QDoubleSpinBox" name="x_step_sb">
          <property name="minimum">
           <double>-100.000000000000000</double>
          </property>
          <property name="maximum">
           <double>100.000000000000000</double>
          </property>
          <property name="value">
           <double>0.100000000000000</double>
          </property>
         </widget>
        </item>
        <item row="4" column="1" colspan="2">
         <widget class="QDoubleSpinBox" name="x_max_sb">
          <property name="minimum">
           <double>-100.000000000000000</double>
          </property>
          <property name="value">
           <double>2.000000000000000</double>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;X_min&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QComboBox" name="functions">
          <property name="currentText">
           <string>sin^2(z) - cos^2(x)</string>
          </property>
          <item>
           <property name="text">
            <string>sin^2(z) - cos^2(x)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>sin(x) * cos(z)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Однопол. гиперболоид</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="11" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Z_step&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Z_min&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;X_max&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>929</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <action name="actionshift">
   <property name="text">
    <string>ЛКМ - добавить точку</string>
   </property>
   <property name="font">
    <font>
     <family>Apple Braille</family>
    </font>
   </property>
  </action>
  <action name="action_4">
   <property name="text">
    <string>Движение мышки + ЛКМ - кривая линия</string>
   </property>
  </action>
  <action name="actionShift">
   <property name="text">
    <string>Shift + ЛКМ - прямая линия</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>ПКМ - замкнуть фигуру</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>ЛКМ - добавить точку</string>
   </property>
  </action>
  <action name="actionshift_2">
   <property name="text">
    <string>shift + ЛКМ - добавить прямую линию</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Движение мышки + ЛКМ - добавить кривую линию</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>ПКМ - замкнуть фигуру</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
